src/h3lib/include/h3api.h.in

```diff
@@ -225,6 +225,7 @@ DECLSPEC H3Error H3_EXPORT(cellToBoundary)(H3Index h3, CellBoundary *gp);
  * Functions for gridDisk
  * @{
  */
+// TODO: Add H3Error
```

Comment:
`maxGridDiskSize` is the last function to add error returns to. One thing that holds it up is looking into changing its output type to be int64_t. I think handling this one requires a separate PR.